stake-pool/cli/src/output.rs

```diff
@@ -0,0 +1,179 @@
+use std::fmt::{Display, Formatter};
+use solana_cli_output::{QuietDisplay, VerboseDisplay};
+use solana_sdk::pubkey::Pubkey;
+use solana_sdk::stake::state::Lockup;
+use serde::{Serialize, Deserialize};
+use spl_stake_pool::state::{Fee, StakePool, StakeStatus, ValidatorList, ValidatorStakeInfo};
```

Comment:
```suggestion
use {
    std::fmt::{Display, Formatter},
    solana_cli_output::{QuietDisplay, VerboseDisplay},
    solana_sdk::{pubkey::Pubkey, stake::state::Lockup},
    serde::{Serialize, Deserialize},
    spl_stake_pool::state::{Fee, StakePool, StakeStatus, ValidatorList, ValidatorStakeInfo},
};
```

Reply:
Looks like the style in this crate is for a single use statement

Reply:
Thx Jon but I don't really understand what you mean. Could you elaborate a bit please? I am newbie in Rust ;-)

Reply:
Hi, I'm not Jon, I'm Tyera. You tagged me on Discord :)
If you look above, I made the desired change as a github "suggestion": only one `use`, instead of 6. Does that help?